plugins/osdn/node.go

```diff
@@ -0,0 +1,140 @@
+package osdn
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	log "github.com/golang/glog"
+
+	"github.com/openshift/openshift-sdn/pkg/netutils"
+	"github.com/openshift/openshift-sdn/plugins/osdn/api"
+
+	osclient "github.com/openshift/origin/pkg/client"
+	osapi "github.com/openshift/origin/pkg/sdn/api"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/container"
+	kexec "k8s.io/kubernetes/pkg/util/exec"
+	kubeutilnet "k8s.io/kubernetes/pkg/util/net"
+)
+
+type OsdnNode struct {
+	multitenant        bool
```

Comment:
I thought we could use pluginName for logging instead of saying single-tenant/multi-tenant plugin but looks like we are already logging pluginName as part of NewNodePlugin(). I'm fine with the current code. Since this is internal stuff we can store pluginName later when needed.